java/vector/src/main/java/org/apache/arrow/vector/UInt1Vector.java

```diff
@@ -62,6 +64,23 @@ public MinorType getMinorType() {
    |          vector value retrieval methods                        |
    |                                                                |
    *----------------------------------------------------------------*/
+  /**
+   * Given a data buffer, get the value stored at a particular position
+   * in the vector.
+   *
+   * <p>To avoid overflow, the returned type is one step up from the signed
+   * type.
+   *
+   * <p>This method should not be used externally.
```

Comment:
i am not sure what this means :) why is the method public but not usable externally..

Reply:
this was copy and paste the real intent was only to use for integration tests.  I've updated it.